### PR TITLE
Cleaning up bower.json ignore and adding .scss-lint

### LINKED ISSUE #91: Expose scss-lint config via bower
The .scss-lint.yml config file should be included in the files that are published via bower. This will allow projects that use scally to share the same linting config.

### PR DESCRIPTION
**Pull request by chris-pearce**
This closes #91.

### CODE REVIEW

File: .scss-lint.yml [hunk 1]
```diff
@@ -1,71 +1,162 @@
...
     force_nesting: true
 
+  # https://github.com/causes/scss-lint/blob/master/lib/scss_lint/linter/README.md#nameformat
   NameFormat:
     enabled: true
-    convention: hyphenated_lowercase # or 'BEM', or a regex pattern
+    convention: BEM
+
+
+
```
Comment (AlecRaeside) [inline .scss-lint.yml]: Extra whitespace?

**Comment by AlecRaeside** [comment]
LGTM apart from whitespace

**Comment by kllevin** [comment]
LGTM

**Comment by chris-pearce** [comment]
This is still a WIP, basically I got into the `.scss-lint` stuff and was like "wow" this is great! So I'm basically beefing it up which will help us capture the trivial issues around code formatting/whitespace. And It'll be part of the Rake task.

**Comment by chris-pearce** [comment]
And most of this is our CSS Guidelines: https://github.com/causes/scss-lint/blob/master/lib/scss_lint/linter/README.md.

**Comment by AlecRaeside** [comment]
LGTM

**Comment by chris-pearce** [comment]
This closes #116.

**Pull request merged**